### ISSUE TITLE
[SF][MessageBox][A11y][Expect Within Two Weeks]Repetitive announcement of MessageBox content in NVDA 2021

### ISSUE THREAD

**Issue by ksblklu** [SAP/ui5-webcomponents-react#3375]
**Describe the bug**
MessageBox content read twice by NVDA 2021

**To Reproduce**
1. Go to [MessageBox document](https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-messagebox--default-story)
2. Click on "open messagebox" button
3. Inspect the speech viewer of NVDA, see the content of MessageBox is read twice.
<img width="1650" alt="image" src="https://user-images.githubusercontent.com/27538066/184838071-39c1e66d-7e37-41e9-a044-eb4d4da76b96.png">

**Expected behavior**
Content of MessageBox should read only once.

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

Expect fix within Two Weeks

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.6.0
`@ui5/webcomponents-react` version: 0.27.3
Operating System: Windows
Browser: Chrome

**Comment by MarcusNotheis** [comment]
Thanks for reporting! 
I think this is an issue with NVDA, even the [SAPUI5 Message Box](https://ui5.sap.com/#/entity/sap.m.MessageBox/sample/sap.m.sample.MessageBox) is announce the Message Box content twice:
<img width="496" alt="image" src="https://user-images.githubusercontent.com/8775211/186937133-94bb4746-e848-4bfc-9760-61783a98b618.png">

With JAWS 2022, the announcement is working as expected:
![image](https://user-images.githubusercontent.com/8775211/186937184-cc49628c-e3c4-4f90-a947-7ebce180facb.png)

**Comment by don-obrien** [comment]
We should not be putting the content, like "Press "Escape" to close the MessageBox." as part of the aria-label.  
From my conversations with central A11y, when a dialog opens, we should be hearing 1) the announcement that this a modal dialog, and 1) the label, which in this case should be "confirmation".  we can set the focus anywhere on a focusable element (which would also read 3) on focus).  the user would know to navigate with virtual cursor to find the rest of the internal contents.  e.g. the part that reads "Press "Escape" to close the MessageBox." should not be immediately read.

**Comment by MarcusNotheis** [comment]
Thanks for your feedback. I've opened an upstream issue https://github.com/SAP/ui5-webcomponents/issues/5781 at the UI5 Web Components repository because we're using the `ui5-dialog` component under the hood.

**Comment by MarcusNotheis** [comment]
Blocked by https://github.com/SAP/ui5-webcomponents/issues/5796
